packages/blade/src/components/Box/Box.stories.tsx

```diff
@@ -97,4 +99,29 @@ AsSection.args = {
   as: 'section',
 } as BoxProps;
 
+export const WithRef = (args: BoxProps): JSX.Element => {
+  const ref: React.Ref<BoxRefType> = React.useRef(null);
+
+  return (
+    <Box height="300px" overflow="auto" backgroundColor="surface.background.level2.lowContrast">
+      <Button
+        onClick={() => {
+          if (!isReactNative()) {
+            castWebType(ref.current).scrollIntoView();
+          }
+        }}
+      >
+        Click to Scroll
+      </Button>
+      <Box ref={ref} {...args}>
+        <Text>Hi from Box with ref</Text>
+      </Box>
+    </Box>
+  );
+};
+
+WithRef.args = {
+  marginTop: '800px',
```

Comment:
Let just put it directly on the box itself? does it need to be configurable?

Reply:
yes I mean this is what we follow for stories so it correct shows up in controls otherwise it shows blank in controls but has value in actual story